app/models/User.js

```diff
@@ -18,7 +20,15 @@ let schema = mongoose.Schema({
 
     email: {
         type: String,
+        required: true,
+        unique: true,
+        trim: true,
         lowercase: true,
+        validate(value) {
+          if (!validator.isEmail(value)) {
+            throw new Error('Invalid email');
```

Comment:
We should also implement a mechanism to prevent bot emails from signing up (e.g. mailinator.com)